### ISSUE TITLE
Evaluate more model architectures

### ISSUE THREAD

**Issue by aazuspan** [lemma-osu/naip-cnn#3]
There are a lot of ways we can build a CNN regression model, so we should experiment with other architectures, batch normalization, dropout, depth, CNN dimensionality, etc.

Models can be added to the `models` module and imported into the `training` notebook to evaluate them.
